app/assets/stylesheets/components/_i18n-dropdown.scss

```
@@ -34,3 +34,12 @@
   position: absolute;
   width: 194px;
 }
+
+.language-desktop-button {
+  background-color: $navy;
```

It looks like this background color is preventing the focused `$blue` coloring from being visible. I think we'd either want to set this to...

```suggestion
  background-color: transparent;
```

...or update the focused styles to apply on the button, rather than on the `.i18n-desktop-toggle`.